rust/operator-binary/src/main.rs

```diff
@@ -224,3 +250,21 @@ async fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+fn references_authentication_class(
+    authentication_config: &Option<SupersetClusterAuthenticationConfig>,
```

Comment:
Nit: Generally, `Option<&T>` is preferred over `&Option<T>`. Not a huge concern for a private function like this, but worth keeping in mind for the future.

Reply:
Thanks for the hint!